src/components/PeopleFilters.tsx

```diff
@@ -16,6 +48,11 @@ export const PeopleFilters = () => {
             type="search"
             className="input"
             placeholder="Search"
+            value={searchParams.get('query') || ''}
+            onChange={e => setSearchParams(getSearchWith(
+              searchParams,
+              { query: e.target.value || null },
+            ))}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)